6-anthos-install/README.md

```diff
@@ -291,6 +291,31 @@ Create a secret with your private key in both clusters.
 
     kubectl apply --context=${CTX_2} -f ${HOME}/terraform-example-foundation-app/6-anthos-install/acm-configs/config-management-west.yaml
 
+### Install Nomos
+Nomos will be installed to know if the Config Management System changes were successfully applied.
```

Comment:
I think we can link to the upstream docs to install nomos (so we don't have to keep install instructions upto date)

Reply:
It is a good idea.